guarddog/analyzer/metadata/potentially_compromised_email_domain.py

```diff
@@ -69,31 +35,35 @@ def detect(self, package_info, path: Optional[str] = None, name: Optional[str] =
 
         emails = self.get_email_addresses(package_info)
 
-        if len(emails) == 0:
+        if len(emails) == 0:  # type: ignore # caused by asbtract method
```

Comment:
`get_email_addresses` should always return a list of strings.
regardless of the abstract.

Reply:
fixed in 289c47e